### PR TITLE
Traitor tele implant

### PR DESCRIPTION
**Pull request by monster860**
This adds a new 5TC traitor item: the teleport implant.

Made from components salvaged from a stolen nuke disk, this implant will, upon use, teleport the user to a random location on the station.

:cl: monster860
rscadd: Adds the teleporter implant, a 5TC traitor item made from components salvaged from a nuke disk that teleports the user to a random location on-the station.
/:cl:

### CODE REVIEW

**Comment by theColdflame** [comment]
slippery antags aren't fun for anyone

**Comment by Regen1** [comment]
Considering this is a complete get out of jail free card it should be more expensive.

For balance purposes it should be debated if it should be unusable if you're cuffed / stunned / incapacitated 

Definitely should be more expensive than the escape implant though

**Comment by Krausus** [comment]
... isn't this literally just #4897, but reposted instead of reopened?

File: code/game/objects/items/weapons/implants/implant_misc.dm [hunk 1]
```diff
@@ -63,4 +63,36 @@
...
+	uses--
+	imp_in.visible_message("<span class='warning'>[imp_in] disappears in a flash of light!</span>", "<span class='warning'>You activate [src], teleporting you to a random location</span>")
+	imp_in.forceMove(get_turf(pick(blobstart)))
+
+/obj/item/weapon/implant/teleport/emp_act(severity)
+	if(uses < 1)
+		return
+	uses--
+	imp_in.visible_message("<span class='warning'>[imp_in] disappears in a flash of light!</span>", "<span class='warning'>[src] malfunctions, teleporting you!</span>")
+	imp_in.forceMove(locate(rand(1,255), rand(1,255), 1)) // Teleport user to random location on z-1
```
Comment (lordpidey) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: Better idea.  Have it drop all worn equipment, and then teleport them to a bloblocation.

**Comment by jughu** [comment]
whats the point of the implant if it drops all your stuff and you cant even get out of maint

**Comment by lordpidey** [comment]
@jughu I'm saying for the EMP act, not the standard activation.

**Comment by jughu** [comment]
Ah then it would be great yes or even better mutations!

**Comment by Aurorablade** [comment]
this isn;t a good idea...like the last time you tried to do this PR. For various balance reasons.

**Comment by jughu** [comment]
Just make bluespace mishappens everytime aslong you have this implant so teleporters/hand teles are a very bad idea since bluespace+bluespace=chaos

**Comment by monster860** [comment]
The purpose of this is to prevent "all antags caught in 5 minutes, this is an extended round

**Comment by theColdflame** [comment]
maybe the antags shouldn't get caught in five minutes then?

**Comment by Krausus** [comment]
> The purpose of this is to prevent "all antags caught in 5 minutes, this is an extended round

That's not a problem you can fix in the code, regardless of how many get-out-of-jail-free cards you add. You simply cannot code away lousy antagonist players. We *do* have admins who can spice up a boring round if they deem it necessary, though.

> Adds the teleporter implant, a 5TC traitor item made from components salvaged from a nuke disk that teleports the user to a random location on-the station.

Same price as a freedom implant, one less use, *way* more actual usefulness as an escape mechanic.

**Comment by IcyV** [comment]
Without a doubt it needs a price bump. Even at one use it's a pretty powerful tool for anyone that can get their hands on it.

**Comment by jughu** [comment]
12 TC is fine

**Comment by tristan1333** [comment]
12 TC will, with a random teleport, be too costly. It'll end up being just flat out not worth the price since it'll probly fail most of the time and just leaves too little other tools to play with. Unless you're a memechem scientist I guess.

**Comment by jughu** [comment]
its not about its not worth it its about its a free jail card when you want

**Comment by tristan1333** [comment]
I know this was closed and is irrelevant, but @jughu that's half your TC's for something which will very likely place you somewhere counter productive.

**Pull request closed without merging**